### DISCUSSION TITLE
Can this be considered in a future feature ?

### DISCUSSION THREAD

**Discussion by jaycraftai** [lllyasviel/Fooocus#3243] [Ideas]
Have you heard about the relight for AI ?

same as this one ?

https://gen-ai.cloud/magnific-relight/

this will be very beneficial ?

hope to hear from you soon ?

**Comment by mashb1t** [comment]
see
* https://github.com/lllyasviel/IC-Light
* https://github.com/lllyasviel/Fooocus/discussions/2888
* https://github.com/lllyasviel/Fooocus/issues/2902#issuecomment-2106253160
